### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by kazim05** [cong235/yolov3_reimplementation#1]
When the initial and end learning rate change(like 1e-4, 1e-6) then the loss goes to Nan even in 2nd STEP and when the initial and end learning rate is same (like 1e-4, 1e-4) then training fine but loss goes to Nan at 5760 STEP. i am using resnet 34 as backbone. please help me to resolve it.

**Comment by uname0x96** [comment]
> When the initial and end learning rate change(like 1e-4, 1e-6) then the loss goes to Nan even in 2nd STEP and when the initial and end learning rate is same (like 1e-4, 1e-4) then training fine but loss goes to Nan at 5760 STEP. i am using resnet 34 as backbone. please help me to resolve it.

Sorry for the late response. I dropped this project but I will up 1 new git Yolov3 + Resnet + DIOU.
About nan in loss, you can keep training it and stop when you take the loss is the lowest. With a small dataset sometime the loss too high and distance between it too big so it makes a nan loss.
